project/src/main/java/com/max/grpc/project/EchoClient.java

```diff
@@ -0,0 +1,45 @@
+
+package com.max.grpc.project;
+
+import com.max.grpc.protos.EchoEntity;
+import com.max.grpc.protos.EchoServiceGrpc;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EchoClient {
+    private static final Logger logger = Logger.getLogger(EchoClient.class.getName());
+
+    private final ManagedChannel channel;
+    private final EchoServiceGrpc.EchoServiceBlockingStub blockingStub;
+
+    public EchoClient(String host, int port) {
+        this(ManagedChannelBuilder.forAddress(host, port)
+            .usePlaintext()
+            .build());
+    }
+
+    EchoClient(ManagedChannel channel) {
+        this.channel = channel;
+        blockingStub = EchoServiceGrpc.newBlockingStub(channel);
+    }
+
+    public EchoEntity sendEcho(int code, String message) throws StatusRuntimeException {
+        logger.info("Trying to send echo message");
+        EchoEntity request = EchoEntity.newBuilder().setCode(123).setMessage(message).build();
+        EchoEntity response = blockingStub.sendEcho(request);
+        //logger.info("Request sent");

```

Comment:
Why do you comment logs?

Reply:
Please look the next two lines. I think it's better to show received data instead of informing about sent request after we have already received the response.